packages/remix-architect/server.ts

```diff
@@ -69,13 +69,21 @@ export function createRemixRequest(
   let host = event.headers["x-forwarded-host"] || event.headers.host;
   let search = event.rawQueryString.length ? `?${event.rawQueryString}` : "";
   let url = new URL(event.rawPath + search, `https://${host}`);
-
+  let isFormData = false;
+  if (
+    event.headers["content-type"] &&
+    event.headers["content-type"].includes("multipart/form-data")
+  ) {
+    isFormData = true;
+  }
```

Comment:
This can be simplified to

```suggestion
  let isFormData = event.headers["content-type"]?.includes("multipart/form-data");
```

Reply:
Ah yes, of course. I had it that way originally and then switched it out to the longer form to produce an easier to follow example .js file for the issue. 

Much better like this, thanks